diff --git a/src/Persistence/Initialization/Version075/MerchantStores.cs b/src/Persistence/Initialization/Version075/MerchantStores.cs
--- a/src/Persistence/Initialization/Version075/MerchantStores.cs
+++ b/src/Persistence/Initialization/Version075/MerchantStores.cs
@@ -140,5 +140,5 @@
             this.ItemHelper.CreateWeapon(72, ItemGroups.Swords, 0, 2, 1, true, false, null),  // Kris             +2+4+L
             this.ItemHelper.CreateWeapon(73, ItemGroups.Swords, 6, 3, 1, true, true, null),   // Gladius          +3+4+L+S
-            this.ItemHelper.CreateWeapon(73, ItemGroups.Swords, 7, 3, 1, true, true, null),   // Falchion         +3+4+L+S
+            this.ItemHelper.CreateWeapon(76, ItemGroups.Swords, 7, 3, 1, true, true, null),   // Falchion         +3+4+L+S
             this.ItemHelper.CreateWeapon(74, ItemGroups.Swords, 8, 3, 1, true, false, null),  // Serpent Sword    +3+4+L
             this.ItemHelper.CreateWeapon(75, ItemGroups.Swords, 5, 3, 1, true, true, null),   // Blade            +3+4+L+S
diff --git a/src/Persistence/Initialization/VersionSeasonSix/MerchantStores.cs b/src/Persistence/Initialization/VersionSeasonSix/MerchantStores.cs
--- a/src/Persistence/Initialization/VersionSeasonSix/MerchantStores.cs
+++ b/src/Persistence/Initialization/VersionSeasonSix/MerchantStores.cs
@@ -141,5 +141,5 @@
             this.ItemHelper.CreateWeapon(72, ItemGroups.Swords, 0, 2, 1, true, false, null),  // Kris             +2+4+L
             this.ItemHelper.CreateWeapon(73, ItemGroups.Swords, 6, 3, 1, true, true, null),   // Gladius          +3+4+L+S
-            this.ItemHelper.CreateWeapon(73, ItemGroups.Swords, 7, 3, 1, true, true, null),   // Falchion         +3+4+L+S
+            this.ItemHelper.CreateWeapon(76, ItemGroups.Swords, 7, 3, 1, true, true, null),   // Falchion         +3+4+L+S
             this.ItemHelper.CreateWeapon(74, ItemGroups.Swords, 8, 3, 1, true, false, null),  // Serpent Sword    +3+4+L
             this.ItemHelper.CreateWeapon(75, ItemGroups.Swords, 5, 3, 1, true, true, null),   // Blade            +3+4+L+S
